tez-mapreduce/src/main/java/org/apache/tez/mapreduce/committer/MROutputCommitter.java

```diff
@@ -119,6 +119,7 @@ public void abortOutput(VertexStatus.State finalState) throws IOException {
         || jobConf.getBoolean("mapred.mapper.new-api", false))  {
       newApiCommitter = true;
     }
+    jobConf.set(MRJobConfig.MR_PARENT_JOB_ID, new org.apache.hadoop.mapred.JobID(String.valueOf(getContext().getApplicationId().getClusterTimestamp()), getContext().getApplicationId().getId()).toString());
```

Comment:
Can we have` String.valueOf(getContext().getApplicationId().getClusterTimestamp()), getContext().getApplicationId().getId()).toString(` inside a method and re-use it in MROutput.java as well ?